src/util/decoders.js

```diff
@@ -218,12 +219,12 @@ export async function constructDataFromBytes(bytes) {
           if (e) {
             throw new Error(e);
           } else {
-            throw new Error('we cannot handle the payload: ', bytes);
+            throw new Error('Error: we cannot handle this payload: ', bytes);
           }
         }
         break;
       default:
-        throw new Error('we cannot handle the payload: ', bytes);
+        throw new Error('we cannot handle the pasta: ', bytes);
```

Comment:
the pasta :) ?
Not suggesting anything since I'm not entirely sure what you wanted to change here.

Reply:
> Could you add the instructions to test it?

I will updating the heroku app with more test cases for this. FYI, the calls-only metadata from kusama v7 is 26 or so frames. It could have been as low as 13 but couldn't get the camera to scan at 2048 frame size or anything above 1024. https://github.com/polkadot-js/ui/issues/224#issuecomment-539438539

> the pasta :) ?

ahah I forgot about this, just changed it to see which block was throwing. 

> We somehow need to reset the isMultipart and/or frameCount at component unmount 

+1. I kept that whilst testing the 26 frames as losing that progress can be devastating, but
I'm thinking of adding an alert that asks whether you want to save your progress or not, if over 2 frames have been scanned. As Jaco mentioned most payloads will be in 2 or less frames, so that would only apply to metadata in practice.

Reply:
>I'm thinking of adding an alert that asks whether you want to save your progress or not, if over 2 frames have been scanned. As Jaco mentioned most payloads will be in 2 or less frames, so that would only apply to metadata in practice.

:+1: